projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -31,6 +33,12 @@ const Index = () => {
 
 	const premiumFeatures = usePremiumFeatures();
 
+	// When page cache is enabled, page cache error needs to be invalidated,
+	// so we can get the updated error message from the last setup run.
+	const invalidatePageCacheError = () => {
+		invalidateQuery( 'page_cache_error' );
+	};
```

Comment:
Only `stores/page-cache.ts` should know what the cache key / datasync key is for the page cache error datastore. Maybe move this to a function in `stores/page-cache.ts`.

Reply:
Oh good catch! I tried a bunch of things and I guess I forgot to move that 😅